ftl/common/cache.py

```diff
@@ -19,13 +19,19 @@
 import datetime
 
 from containerregistry.client import docker_name
+from containerregistry.client import docker_creds
 from containerregistry.client.v2_2 import docker_image
 from containerregistry.client.v2_2 import docker_session
 
 from ftl.common import ftl_util
 
 _DEFAULT_TTL_WEEKS = 1
 
+_GLOBAL_CACHE_REGISTRY = 'gcr.io/ftl-global-cache'
+
+# TODO(nkubala): enable feature flag for global cache
+_USE_GLOBAL_CACHE = False
```

Comment:
does it make sense for this to be toggled at the binary level or would it make more sense as be a flag option? (I think your comment is saying the same thing)

Reply:
yeah good call. changed this to be a flag, defaults to false so the cache is opt-in right now. once we want to release we can either remove the flag or at the very least change the default value.